src/GraphQL.ApiTests/GraphQL.NewtonsoftJson.approved.txt

```diff
@@ -44,6 +44,8 @@ namespace GraphQL.NewtonsoftJson
         public static T FromJson<T>(this string json) { }
         [System.Obsolete("This method will be removed in a future version of GraphQL.NET.")]
         public static object GetValue(this object value) { }
+        [System.Obsolete("This class will be removed in a future version of GraphQL.NET. Please use the ToI" +
```

Comment:
```suggestion
        [System.Obsolete("This method will be removed in a future version of GraphQL.NET. Please use the ToI" +
```

Reply:
This has been updated.  On my PC it still shows it unchanged on GitHub's "files changed" tab (!!), but viewing the file on GitHub shows that it has been properly updated.